pkg/operator/operator/k8s.go

```diff
@@ -369,10 +384,22 @@ func pythonPredictorContainers(api *spec.API, envVars []kcore.EnvVar) ([]kcore.C
 }
 
 func TensorFlowPredictorContainers(api *spec.API) ([]kcore.Container, []kcore.Volume) {
-	return tensorFlowPredictorContainers(api, getEnvVars(api, APIContainerName))
+	envVars := getEnvVars(api, APIContainerName)
+	tfServingEnvVars := getEnvVars(api, _tfServingContainerName)
+	return tensorFlowPredictorContainers(api, envVars, tfServingEnvVars, false)
 }
 
-func tensorFlowPredictorContainers(api *spec.API, envVars []kcore.EnvVar) ([]kcore.Container, []kcore.Volume) {
+func TensorFlowPredictorJobContainers(api *spec.API) ([]kcore.Container, []kcore.Volume) {
+	envVars := getEnvVars(api, APIContainerName)
+	envVars = append(envVars, getKubexitEnvVars(APIContainerName)...)
+
+	tfServingEnvVars := getEnvVars(api, _tfServingContainerName)
+	tfServingEnvVars = append(tfServingEnvVars, getKubexitEnvVars(_tfServingContainerName, APIContainerName)...)
```

Comment:
Should we also add `_neuronRTDContainerName` to `getKubexitEnvVars` as a param? If `compute.inf` > 0 and the tensorflow predictor is used, then the `_neuronRTDContainerName` will also be present as a sidecar.

Reply:
makes sense